code/modules/mob/living/simple_animal/friendly/drone/interaction.dm

```diff
@@ -115,8 +115,8 @@
 		return
 	if(hack)
 		if(hacked)
-			return
 			Stun(40)
```

Comment:
This should probably just be unindented.

Reply:
I'm unsure what the intended behavior is here so I'm just guessing

Reply:
The hacked check looks like it's supposed to be an early return and the stun is for when they first get hacked